servers/quarkus-server/src/main/resources/application.properties

```diff
@@ -19,6 +19,29 @@
 nessie.server.default-branch=main
 nessie.server.send-stacktrace-to-client=false
 
+## Nessie authorization settings
+### This will perform authorization on branches/tags and contents where rule definitions are
+### using a Common Expression Language (CEL) expression (an intro to CEL can be found at https://github.com/google/cel-spec/blob/master/doc/intro.md).
+### Rule definitions are of the form nessie.server.authorization.rules.<ruleId>="<rule_expression>"
+### Available variables within the <rule_expression> are: 'op' / 'role' / 'ref' / 'path'
+### The 'op' variable in the <rule_expression> can be any of:
+### 'CREATE_REFERENCE', 'DELETE_REFERENCE', 'READ_ENTRIES', 'LIST_COMMIT_LOG', 'COMMIT_CHANGE_AGAINST_REFERENCE', 'ASSIGN_REFERENCE_TO_HASH', 'UPDATE_ENTITY', 'READ_ENTITY_VALUE', 'DELETE_ENTITY'
+### The 'role' refers to the user's role and can be any string
+### The 'ref' refers to a string representing a branch/tag name
+### The 'path' refers to the Key for the contents of an object and can be any string
+### Some example rules are shown below:
+# nessie.server.authorization.enabled=false
+# nessie.server.authorization.rules.allow_branch_creation="op=='CREATE_REFERENCE' && role.startsWith('test_user') && ref.startsWith('allowedBranch')"
```

Comment:
Can you change these examples to some say "use case based" examples - like "allow all too all users named 'admin'" or "allow only reads to all users names 'foo'"?
Having one expression per access-check is probably a bad example, because it's not necessary to do so.

Reply:
done